src/training_hub/profiling/memory_estimator.py

```
@@ -689,77 +705,54 @@ def __init__(
+                batch_size: int | None = None,
+                max_seq_len: int | None = None,
+                max_tokens_per_gpu: int | None = None,
+                use_liger: bool = True,
+                verbose: int = 1,
+                trust_remote_code: bool = False,
+                unfreeze_rank_ratio: float = 0.25,
+                ):
+        reason = "All experimental functionality has been moved into the main OSFTEstimator class, " + \
+            "please use that instead.\nIf you are seeing this by using the 'osft-e' option in the " + \
+            "estimate function, please use 'osft' instead."
+        warnings.warn(reason, category=warnings.DeprecationWarning)
```

Ruff (0.15.2)

[warning] No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)